frontend/src/concepts/distributedWorkloads/DistributedWorkloadsContext.tsx

```diff
@@ -25,6 +26,7 @@ type DistributedWorkloadsContextType = {
   projectCurrentMetrics: DWProjectCurrentMetrics;
   refreshAllData: () => void;
   namespace?: string;
+  projectDisplayName?: string;
```

Comment:
Not optional -- you can always give an answer

Reply:
Ah --- I remember now why these were optional to begin with. In the initial design for Distributed Workload Metrics we had our Cluster Metrics page which was not namespaced, so the context actually legitimately could have no namespace selected. Hopefully this is the last vestige of that we missed.